src/ngraph/runtime/intelgpu/intelgpu_op_custom_kernels.cpp

```diff
@@ -1519,6 +1519,191 @@ void runtime::intelgpu::do_reshape_operation(cldnn::topology& topology,
     topology.add(op_reshape);
 }
 
+void runtime::intelgpu::do_quantize_operation(cldnn::topology& topology,
+                                              const string& input0_name,
+                                              const Shape& input0_shape,
+                                              const element::Type& input0_type,
+                                              const string& input1_name,
+                                              const Shape& input1_shape,
+                                              const string& input2_name,
+                                              const Shape& input2_shape,
+                                              const string& output_name,
+                                              const Shape& output_shape,
+                                              const element::Type& output_type,
+                                              const AxisSet& axis,
+                                              const ngraph::op::Quantize::RoundMode mode)
+{
+    const cldnn::layout layout = IntelGPULayout::create_cldnn_layout(output_type, output_shape);
+    const string entry_point_name = "quantize_" + output_name;
+    const string real_type_str = get_opencl_type_name(input0_type);
+    const string quant_type_str = get_opencl_type_name(output_type);
+    codegen::CodeWriter writer;
+    vector<size_t> gws;
+
+    gen_func_def(writer,
+                 entry_point_name,
+                 {real_type_str, real_type_str, quant_type_str},
+                 {input0_shape, input1_shape, input2_shape},
+                 quant_type_str,
+                 output_shape);
+
+    writer.block_begin();
+    {
+        writer << "// " << axis << "\n"
+               << "// rounding mode: " << (int)mode << "\n";
+
+        // Main loops
+        gws = generate_loops(writer, input0_shape, true);
+
+        // apply scale
+        writer << real_type_str << " qvalue = input0" << access_dims(input0_shape) << " / input1"
+               << access_dims(input1_shape) << ";\n";
+
+        // round
+        switch (mode)
+        {
+        case ngraph::op::Quantize::RoundMode::ROUND_NEAREST_TOWARD_INFINITY:
+        {
+            writer << real_type_str << " abs_qvalue = fabs(qvalue);\n"
+                   << real_type_str << " abs_qvalue_toward_inf = floor(abs_qvalue + 0.5);\n"
+                   << "qvalue = (qvalue < 0.0) ? -abs_qvalue_toward_inf : abs_qvalue_toward_inf;\n";
+            break;
+        }
+        case ngraph::op::Quantize::RoundMode::ROUND_NEAREST_TOWARD_ZERO:
+        {
+            writer
+                << real_type_str << " abs_qvalue = fabs(qvalue);\n"
+                << real_type_str << " abs_qvalue_toward_zero = ceil(abs_qvalue - 0.5);\n"
+                << "qvalue = (qvalue < 0.0) ? -abs_qvalue_toward_zero : abs_qvalue_toward_zero;\n";
+        }
+        break;
```

Comment:
You should be consistent where you put the break; statement. In the previous case you put it inside the brackets while for this case you put it outside the brackets. For consistency with the rest of the code please put break; inside brackets. This applies to all cases here.

Reply:
@rkimballn1 Good catch! Usually I put the `break` after brackets but it depends on the project code style.  This particular statement placement was a typo.